### DISCUSSION TITLE
MeshAgent crashes on frist User Consent

### DISCUSSION THREAD

**Discussion by MaChu1284** [Ylianst/MeshCentral#5959] [Issues]
Hello,

I have an issue with all my clients (12 Windows 11 Clients). They are all configured to ask for user consent (Device Group setting).
When I'm on the Server and go to a device -> Desktop -> Connect the user consent window pops up as expected on the client, but clicking on it (doesnt matter if its the button or just the windows) causes it to not respond and finally crash and close.
After that I can reinitiate the Desktop connection, User Consent pops up and works perfectly. This is until I reboot the Machine with the agent on it.

The behaviour is the same on all machines and can be reproduced.

MeshCentral Server is up to date: 1.1.22

Any Ideas what could cause this behaviour?

Thank you!

**Comment by si458** [comment]
ive seen this from time to time but never been able to reproduce it every single time
do you have just consent set OR do you also display a notification OR display the toolbar?
do you have any like apraphies in your users real name?

**Comment by MaChu1284** [comment]
I'm sorry, I was wrong with the version:
I was running 1.1.21 and updated to .22 now, and cant reproduce the behaviour anymore. It seems like the update fixed the issue.

All Users that use the MeshCentral Server are Federated Azure AD Users (which have some cryptic UPN and also special Charachters in the Display Names.
The toolbar is also set.

thank you for the fast response.

**Comment by geekasso** [comment]
I am on **1.1.22** testing it in a Win 11 VM and it consistently crashes and sometimes the consent window does not even show or only shows the box before crashing. 

![image](https://github.com/Ylianst/MeshCentral/assets/7304516/111d72d7-e42c-4682-b028-29fde8e4bbe5)

When it does show, the buttons are not clickable.

![image](https://github.com/Ylianst/MeshCentral/assets/7304516/a2db1529-8814-4ed7-a848-e37880b2919a)

When it does connect, it crashes when dragging the privacy bar. 

Issues persist with 'desktopprivacybar' set to false as well.
